rust_src/remacs-macros/README.md

```diff
@@ -0,0 +1,44 @@
+remacs-macros
+=============
+
+# Attributes
+
+## `lisp_fn`
+This macros creates the necessary functions and symbols automatically.
+It handles `MANY` functions and normal functions too (support for `UNEVALLED`
```

Comment:
I think the note about `UNEVALLED` belongs in a comment rather than in the docs that new users see first.